src/match/match-item.html

```diff
@@ -40,6 +40,15 @@
       @apply --ppaer-font-body1;
       color: var(--secondary-text-color);
     }
+
+    .positive {
+      color: #1fef5a;
```

Comment:
please choose some colors from `paper-styles/colors.html` and use it with `var(...)`